spec.html

```diff
@@ -22644,7 +22644,7 @@ <h1>Runtime Semantics: HostResolveImportedModule ( _referencingScriptOrModule_,
             If a Module Record corresponding to the pair _referencingScriptOrModule_, _specifier_ does not exist or cannot be created, an exception must be thrown.
           </li>
           <li>
-            This operation must be idempotent if it completes normally. Each time it is called with a specific _referencingScriptOrModule_, _specifier_ pair as arguments it must return the same Module Record instance.
+            Each time this operation is called with a specific _referencingScriptOrModule_, _specifier_ pair as arguments it must return the same Module Record instance if it completes normally.
```

Comment:
@tc39/ecma262-editors Please confirm that this changed wording matches the intended normative requirement that the mathematical-function-like behaviour only applies for normal completions. It was unclear whether the qualifier in the previous sentence was meant to apply to this whole `<li>` or whether the two sentences were meant to be read completely independently.

Reply:
(relates somewhat to #1645)

the rewording seems correct to me; cc @domenic to confirm.

Reply:
That was the intent.